shadow-cljs.edn

```diff
@@ -7,8 +7,13 @@
  :dependencies
  [[binaryage/devtools "0.9.7"]
   [devcards "0.2.5"]
+  [cider/cider-nrepl "0.40.0"]
+  [refactor-nrepl/refactor-nrepl "3.9.0"]
   [cljs-bean "1.5.0"]]
 
+ :middleware [cider.nrepl/cider-middleware
+              refactor-nrepl.middleware/wrap-refactor]
+
```

Comment:
shouldn't this be added by the `cider-jack-in` command (or similar in VS Code/etc.)?

Reply:
> shouldn't this be added by the cider-jack-in command (or similar in VS Code/etc.)?

It's added, but a good chunk of users prefer to create a server they can `cider-connect` too